<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RemoveCarWindow</class>
 <widget class="QDialog" name="RemoveCarWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>813</width>
    <height>316</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>0</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>1500</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTableWidget" name="show_disp">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>851</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>1700</width>
       <height>500</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>9</pointsize>
       <stylestrategy>NoAntialias</stylestrategy>
      </font>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::NoContextMenu</enum>
     </property>
     <column>
      <property name="text">
       <string>Disponibilidade</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Matricula</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Marca</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Modelo</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Ano</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Cilindrada</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Potencia</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="bot_menu">
       <property name="text">
        <string>Menu</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLineEdit" name="input_remove">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Inserir Matricula para Remover&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bot_remover">
       <property name="text">
        <string>Remover</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
